Document/0x06i-Testing-Code-Quality-and-Build-Settings.md

```diff
@@ -34,6 +34,29 @@ Sealed Resources version=2 rules=12 files=1410
 Internal requirements count=1 size=176
 ```
 
+### Determining Whether the App is Debuggable
+
+#### Overview
+
+Debugging iOS applications can be done using Xcode, which embeds a powerful debugger called lldb. Lldb is the default debugger since Xcode5 where it replaced GNU tools like gdb and is fully integrated in the development environment. While debugging is a useful feature when developing an app, it has to be turned off before releasing apps to stores.
+
+Generating an app in Build or Release mode depends on build settings in Xcode; when an app is generated in Debug mode, a DEBUG flag is inserted in the generated files.
+
+#### Static Analysis
+
+First, a way to determine the mode in which your app is to be generated to check the flags in the environment:
```

Comment:
To:
> A first way to determine the mode in which your app will be generated is to check the flags in the environment:

Reply:
What if i change it to "At first you need to determine the mode..." ?